src/main/java/ee/developest/gtm/GtmWrapper.kt

```diff
@@ -189,8 +191,16 @@ class GtmWrapper {
 
     }
 
-    private fun initGtm(project: Project): Boolean {
-        val process = ProcessBuilder(gtmExePath, INIT_COMMAND, CWD_OPTION, project.basePath).start();
+    private fun initGtm(project: Project, local: Boolean): Boolean {
+        val arguments = ArrayList<String?>()
```

Comment:
I'd rather have a null check and show some error here, as if gtmExe path is null or project base path is null, it will result in an error anyhow.
Then `ArrayList<String?>` could be changed to `ArrayList<String>`

Reply:
An error will already be thrown if gtmExePath is not found during initialization. Should I "throw" the same error there as well?

Reply:
An error will already be thrown if gtmExePath is not found during initialization. Should I throw the same error there as well?

Reply:
I think I'll just throw a more general error for both of them, like "Paths could not be found, please check the log for more information"

Reply:
Yes, sounds good. Also if there is no chance we can get here due to some previous check just silence nullability warn with `!`